### PR TITLE
Add functional test

### PR DESCRIPTION
**Pull request by franmomu**
Adding a minimal functional test that includes the twig template, it will fail as I'm finding some errors.

### CODE REVIEW

File: tests/App/AppKernel.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+final class AppKernel extends Kernel
+{
+    use MicroKernelTrait;
+
+    public function __construct()
+    {
+        parent::__construct('dev', true);
+    }
+
+    public function registerBundles()
```
Comment (greg0ire) [inline tests/App/AppKernel.php]:
```suggestion
    public function registerBundles(): array
```

File: tests/Functional/FunctionalTest.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+final class FunctionalTest extends WebTestCase
+{
+    public function testRenderFlashes(): void
+    {
+        $kernel = new AppKernel();
+
+        $client = $this->createClientBC($kernel);
+        $client->request('GET', '/');
+
+        var_dump($client->getResponse()->getContent());
```
Comment (greg0ire) [inline tests/Functional/FunctionalTest.php]: Should be removed?

Comment (franmomu) [reply tests/Functional/FunctionalTest.php]: oops 😬

**Comment by greg0ire** [comment]
Thanks @franmomu !

**Pull request merged**